env.io-functions-bonus

```diff
@@ -0,0 +1,34 @@
+COMPOSE_PROJECT_NAME=io-fn-bonus
+
+COSMOSDB_PORT=3000
+API_GATEWAY_PORT=80
+
+FUNCTIONS_WORKER_RUNTIME=node
+
+AzureWebJobsStorage=${STORAGE_CONNECTION_STRING}
+REQ_USER_GROUPS=ApiProfileWrite
+REQ_SERVICE_ID=MyServiceId
+SERVICE_EMAIL=example@email.com
```

Comment:
I think we don't need these values here since it's called directly by the backend

Reply:
Fixed in 661e922